### DISCUSSION TITLE
能否实现在画中画模式播放视频时显示弹幕呢🙇‍♂️🙇‍♂️🙇‍♂️

### DISCUSSION THREAD

**Discussion by RRRRRqing** [the1812/Bilibili-Evolved#4143] [功能建议 / Ideas]
能否实现在画中画模式播放视频时显示弹幕呢🙇‍♂️🙇‍♂️🙇‍♂️

**Comment by the1812** [comment]
可以做一个玩玩, 但需要浏览器支持 [Document Picture-in-Picture](https://wicg.github.io/document-picture-in-picture/) (目前貌似只有 Chrome 用 `chrome://flags/#document-picture-in-picture-api` 可以打开)

**Reply by RRRRRqing** [reply]
谢谢！！辛苦啦👏👏

**Comment by niu541412** [comment]
碰巧看到这个讨论，现在有大佬做了chrome扩展实现了这个功能，https://github.com/apades/dmMiniPlayer

**Reply by RRRRRqing** [reply]
哇太完美了 谢谢！
